### PR TITLE
Change of discontinuation date from text to date

### PR DESCRIPTION
**Pull request by ckote**
Changes to also support the pulling of facility list other than inputting text

### CODE REVIEW

File: configuration/ampathforms/MCH-MS_Discontinuation.json [hunk 1]
```diff
@@ -101,19 +101,12 @@
 							}
 						},
 						{
-							"label": "(if transferred out)Transfer to Facility:",
-							"type": "obs",
+							"label": "Transfer to Facility(if transferred out):",
+							"type": "encounterLocation",
```
Comment (ojwanganto) [inline configuration/ampathforms/MCH-MS_Discontinuation.json:105]: How is this information saved in the database? I know a form can only have one encounter location. @donaldkibet @makombe please help me understand this. Asante

Comment (donaldkibet) [reply configuration/ampathforms/MCH-MS_Discontinuation.json:105]: We should be using the remote-select here and specify the data source to be locations.

**Review by donaldkibet — CHANGES_REQUESTED** [review]
we should use `remote-select` here and specify `location` as the data source.

**Comment by ckote** [comment]
HI @donaldkibet do you mean this:

"questionOptions": {
                "rendering": "remote-select",
                "appointmentKey": "location",
                "dataSource": "location"
              }

**Comment by ckote** [comment]
Without the appointment Key though?

**Comment by ckote** [comment]
This is done

**Pull request merged**